test/integration/progressesTasks.test.js

```diff
@@ -516,4 +516,104 @@ describe("Test Progress Updates API for Tasks", function () {
         });
     });
   });
+
+  describe("GET /progresses (getPaginatedProgressDocument)", function () {
```

Comment:
Can we use arrow function declaration?

Reply:
I'm following the same approach as the other tests in this file, where regular function declarations are used. If necessary, I can switch to arrow functions